src/components/utils/__tests__/redux.test.ts

```diff
@@ -0,0 +1,43 @@
+import { buildCaptureStateKey } from '../redux'
+
+describe('utils', () => {
+  describe('redux', () => {
+    describe('buildCaptureStateKey', () => {
+      it('should return correct state key for document payloads', () => {
+        expect(buildCaptureStateKey({ method: 'document' })).toEqual(
+          'document_front'
+        )
+        expect(
+          buildCaptureStateKey({ method: 'document', side: 'front' })
+        ).toEqual('document_front')
+        expect(
+          buildCaptureStateKey({ method: 'document', side: 'back' })
+        ).toEqual('document_back')
+        expect(
+          buildCaptureStateKey({
+            method: 'document',
+            side: 'back',
+            variant: 'video',
+          })
+        ).toEqual('document_video')
+      })
+
+      it('should return correct state key for face payloads', () => {
+        expect(buildCaptureStateKey({ method: 'face' })).toEqual('face')
+        expect(buildCaptureStateKey({ method: 'face', side: 'front' })).toEqual(
+          'face'
+        )
+        expect(buildCaptureStateKey({ method: 'face', side: 'back' })).toEqual(
+          'face'
+        )
+        expect(
+          buildCaptureStateKey({
+            method: 'face',
+            side: 'back',
+            variant: 'video',
+          })
+        ).toEqual('face')
```

Comment:
Looks like the expected value is all the same. Would be more maintainable if `'face'` is assigned to a variable.

Reply:
There are dedicated tests for `'document'` already: https://github.com/onfido/onfido-sdk-ui/blob/d7335827abb14f1585f559c8e7642b61133a45f0/src/components/utils/__tests__/redux.test.ts#L6-L23

Reply:
Ah, ok. Ignore me then.